main.cpp

```diff
@@ -13,75 +13,127 @@ What you create in this project will be used as the basis of Project 5 in the co
+    Cell Phone
+
+    A Cell Phone is made up of the following 5 properties:
+        Display
+        Memory
+        CPU
+        Radio
+        Applications
+
+    A Cell Phone has 3 things it can do:
+        make a call
+        send a text
+        run an application.
+
+    Notice that I did not use "has a display" or "Has memory" or "has a cpu" as one of the properties of the CellPhone
+    
+    Writing 'has a ___" checks whether or not your object **has the ability to do something**.
+    Instead, I wrote "Display" or "CPU".  These are specific objects or amounts. 
+    
+    In C++ terms, this means to I want you to avoid using 'bool' (has a) as a member variable type.
+    Instead, prefer the other primitive types.
+
+    When you choose your 5 smaller parts, remember that each of these 5 Sub Objects will need 
+    to be defined with 5 primitive properties and 3 actions EACH.  
+
+2) write the name of the primitive type you'll be using after each property in UDTs 1-4:
+    pick properties that can be represented with 'int float double bool char std::string'
+    example: 
+        Display:
+            Number of Pixels (int)
+            Amount of Power consumed (milliwatt-hours) (float)
+            Brightness (double)
+            area in cm2 (int)
+            brand (std::string)
+
+3) Move the Thing 10 pseudo-code to after your Thing1-4 in the project.
 */
+
+/*
+Thing 1) Song
+5 properties:
+    1) duration of the song // double
+    2) tempo of the song // int 
+    3) key of the song // char
+    4) number of artists performing on the song // int
+    5) record label that released the song // string
+3 things it can do:
+    1) played on the radio
+    2) performed live at a concert
+    3) be remixed by another artist
+ */
+
+/*
+Thing 2) Sandwich
+5 properties:
+    1) type of bread // string
+    2) type of fillings // string
+    3) amount of different fillings // int
+    4) condiment used // string
+    5) number of calories // int
+3 things it can do:
+    1) cause foodpoisoning
+    2) provide a satisfying meal
+    3) go moldy if left uneaten
+ */
+
+/*
+Thing 3) Soccer Team
+5 properties:
+    1) number of players in squad // int
+    2) average age of players in squad // int
+    3) number of total goals scored by the team // int
+    4) number of matches in the season // int
+    5) league position // int
+3 things it can do:
+    1) train their skills
+    2) win a game
+    3) move up the league table
+ */
+
+/*
+Thing 4) Synth
+5 properties:
+    1) number of oscillators // int
+    2) amount of presets // int
+    3) number of filters // int
+    4) number of outputs // int
+    5) type of built-in effect // string
+3 things it can do:
+    1) output midi
+    2) filter out all frequences above 10khz
+    3) load different presets
+ */
+
+/*
+Thing 10) Motorbike
+5 properties:
+    1) seat // string
+    2) engine // string
+    3) exhaust // string
+    4) fuel tank // string
+    5) tyre // string
```

Comment:
You need to pick properties that are OBJECTS, not primitives. 
these 5 properties will become Thing 5 - 9 in part 1d.
Remember, each Thing that you define (excluding Thing 10) needs 5 properties and 3 actions.
- [ ] revise